docs-new/docs/use-cases/non-k8s.md

```diff
@@ -96,7 +114,10 @@ You can then apply this YAML file with the following command:
 kubectl apply -f test-task.yaml -n my-keptn-annotated-namespace
 ```
 
-Applying this file causes Keptn to create a Job and a Pod
+Applying this file causes Keptn to create a Kubernetes
+[job](https://kubernetes.io/docs/concepts/workloads/controllers/job/)
```

Comment:
mentioning the Pod might not be needed here, as the creation of a Kubernetes Job implies that a Pod will then be created by the Job controller. But I also don't mind if we keep it to be more explicit

Reply:
I wondered about that myself but you had said it in your reply to Artur so I passed it through.  But I don't think mentioning the pod adds anything here so I deleted it.